Description: Use realpath instead of custom script
 Use realpath instead of the custom script, which is broken. As an example,
 .
   $ ./mk/relpath.sh /usr/lib/ghc/lib /usr/lib/ghc-doc
   ..-doc
   $ realpath --relative-to=/usr/lib/ghc/lib /usr/lib/ghc-doc
   ../../ghc-doc
Author: Ilias Tsitsimpis <iliastsi@debian.org>

Index: b/hadrian/bindist/Makefile
===================================================================
--- a/hadrian/bindist/Makefile
+++ b/hadrian/bindist/Makefile
@@ -232,7 +232,7 @@ update_package_db: install_bin install_l
 	$(INSTALL_DATA) mk/system-cxx-std-lib-1.0.conf "$(DESTDIR)$(ActualLibsDir)/package.conf.d"
 	@echo "Updating the package DB"
 	$(foreach p, $(PKG_CONFS),\
-		$(call patchpackageconf,$(shell echo $(notdir $p) | sed 's/-\([0-9]*[0-9]\.\)*conf//g'),$(shell echo "$p" | sed 's:\0xxx\0:   :g'),$(docdir),$(shell mk/relpath.sh "$(ActualLibsDir)" "$(docdir)"),$(shell echo $(notdir $p) | sed 's/.conf//g')))
+		$(call patchpackageconf,$(shell echo $(notdir $p) | sed 's/-\([0-9]*[0-9]\.\)*conf//g'),$(shell echo "$p" | sed 's:\0xxx\0:   :g'),$(docdir),$(shell realpath -z --relative-to="$(ActualLibsDir)" "$(docdir)"),$(shell echo $(notdir $p) | sed 's/.conf//g')))
 	'$(DESTDIR)$(ActualBinsDir)/$(CrossCompilePrefix)ghc-pkg' --global-package-db "$(DESTDIR)$(ActualLibsDir)/package.conf.d" recache
 
 install_mingw:
